Portal_Perfomance_Employes/Models/Employee.cs

```diff
@@ -1,12 +1,13 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
-using System.ComponentModel.DataAnnotations;
-using System.ComponentModel.DataAnnotations.Schema;
+﻿using System.ComponentModel.DataAnnotations;
 
 namespace PortalPerfomanceEmployees.Models
 {
     public class Employee
```

Comment:
Please add "public int TeamId { get; set; }" 
We had it in pr "feature/add-userInfo-controller #12"